### ISSUE TITLE
cannot sudo startxfce4

### ISSUE THREAD

**Issue by jamesnwright** [dnschneid/crouton#3300]
```sudo startxfce4
Please paste the output of the following command here: sudo edit-chroot -all
```Entering /mnt/stateful_partition/crouton/chroots/kali-rolling...
mv: cannot stat ‘/run/crouton/mnt/stateful_partition/crouton/chroots/kali-rolling/run/dri/card*’: No such file or directory
Unmounting /mnt/stateful_partition/crouton/chroots/kali-rolling...

#### Please describe your issue:
when i try to start it shows this cannot stat error

#### If known, describe the steps to reproduce the issue:

**Comment by DennisLfromGA** [comment]
@jamesnwright,

Can you get into your chroot using: `sudo enter-chroot -n kali-rolling`
If so, then please show us the output of: `croutonversion`

-**DennisL**

**Comment by jamesnwright** [comment]
Entering /mnt/stateful_partition/crouton/chroots/kali-rolling...
mv: cannot stat ‘/run/crouton/mnt/stateful_partition/crouton/chroots/kali-rolling/run/dri/card*’: No such file or directory
Unmounting /mnt/stateful_partition/crouton/chroots/kali-rolling...

i get the same thing with the sudo enter-chroot -n kali-rolling command

**Comment by jamesnwright** [comment]
i could try doing the whole process over again with powerwashing, should I?

**Comment by DennisLfromGA** [comment]
@jamesnwright,

A powerwash is never necessary to remove a crouton chroot, just use: 

`sudo delete-chroot kali-rolling`

*Hope this helps,*
-**DennisL**

**Comment by jamesnwright** [comment]
umm, I deleted it, now what should I do (sorry I honestly have a very vague idea of what I'm doing)

**Comment by DennisLfromGA** [comment]
@jamesnwright,

Don't know, since you installed Kali I assume you want to do [kali things](https://www.kali.org/).

To get past vague, you could take a look at the [README](https://github.com/dnschneid/crouton/blob/master/README.md), it has some pretty good info. & guidance.
There's also the [crouton wiki](https://github.com/dnschneid/crouton/wiki) and a [crouton subreddit](https://www.reddit.com/r/Crouton/)

*Hope this helps,*
-**DennisL**

**Comment by gfctam** [comment]
after you delete the chroot, look at this video https://www.youtube.com/watch?v=qjvD1dBZY4A. or...

sudo sh -e ~/Downloads/crouton -r kali-rolling -n kali -t xiwi
sudo sh -e ~/Downloads/crouton -r kali-rolling -u -n kali -t xiwi
cd /usr/local/chroots/kali/etc/apt
sudo vi sources.list
- Modify those links to become https
sudo sh -e ~/Downloads/crouton -r kali-rolling -u -n kali -t xiwi
sudo enter-chroot -n kali
cd /etc/apt
sudo nano sources.list
- add the following line: deb http://ftp.us.debian.org/debian jessie main
save (ctrl + o) and exit (ctrl + x) nano )
sudo apt-get update
sudo apt-get install netsurf-gtk
exit
sudo sh -e ~/Downloads/crouton -u -n kali -t xfce
sudo startxfce4

- In Kali chroot:
( apt-get install kali-linux-full
Once finished, shut down kali linux )
- If you have this problem -> Only console users are allowed to run the X server
- In crouton shell
sudo enter-chroot -n kali
sudo -i
cd /etc/X11
nano Xwrapper.config
- change the last line "allowed_users=console" to "allowed_users=anybody"
- save (ctrl o) and exit (ctrl x)

- Xfce-power-manager:
apt-get remove xfce4-power-manager
apt-get install build-essential
apt-get build-dep xfce4-power-manager
apt-get install libupower-glib-dev
- Download the new version power manager -  http://git.xfce.org/xfce/xfce4-power-manager/snapshot/xfce4-power-manager-1.4.1.tar.bz2 
- Copy it to /usr/src
cd /usr/src
tar xjf xfce4-power-manager-1.4.1.tar.bz2
cd xfce4-power-manager-1.4.1

./autogen.sh --prefix=/usr --disable-static --enable-network-manager --enable-polkit --disable-debug --enable-lxdepanel
make
make install

**Comment by gfctam** [comment]
close #3300

**Comment by gfctam** [comment]
Close #3300

**Comment by gfctam** [comment]
Closes dnschneid/crouton#3300

**Comment by gfctam** [comment]
All done now!
